azure/notebooks/kubernetes/Detailed_setup_guide.md

```diff
@@ -0,0 +1,213 @@
+
+- Once `az` is installed, make sure you you configure the local `az` cli to work with your Azure credentials, do `az login` and authenticate from Microsoft's website.
+
+- For more details follow the steps [Here](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli). 
+
+
+### [Step 1: Install Kubectl](#anchor-install-kubectl)
+
+- Install `kubectl` to access your cluster from the local machine from the following link : [https://kubernetes.io/docs/tasks/tools/] depending on your operating system. 
+
+
+### [Step 2: Set some environment variables](#anchor-set-env-variables)
+We will set some environment variables beforehand, namely to help us deploy some resources quickly.  We will keep on using some common values for the rest of the deployments. . 
+```bash
+REGION_NAME=<your preferred location>
+RESOURCE_GROUP=<your preferred resource group name>
+SUBNET_NAME=<optional subnet name>
+VNET_NAME=<optional vnet name>
+VERSION=$(az aks get-versions --location $REGION_NAME --query \
+                'orchestrators[?!isPreview] | [-1].orchestratorVersion' \--output tsv)
+AKS_CLUSTER_NAME=<your cluster name>
+VM_SIZE=Standard_NC12s_v3 # or any other VM size. We use VMs with GPU
+```
+
+- We would need a resource group. You can create a resource group in Azure `az group create  --name $RESOURCE_GROUP --location $REGION_NAME`, or use an existing one.
+
+- Secondly, we get the latest non preview Kubernetes version and store it in an env variable using. This provides that the latest version is 1.20.5. Optionally you can directly set the Kubernetes version in the environment variable `VERSION`.
+
+- **NOTE 1**: There are two network modes to choose from when deploying an AKS cluster. The default one is *Kubenet networking* which we will use here. The other more sophisticated one is Azure Container Networking Interface. We will not need this at the moment. Please look [here](https://docs.microsoft.com/en-us/learn/modules/aks-workshop/02-deploy-aks) to get an idea about how to deploy Azure CNI.  
+
+- **NOTE 2**: Depending on your account limitations, the number and type of VMs that you can spin up may vary. Also there may be zone limitations. Make sure you spin up VMs having GPU capabilities GPU: NVIDIA Pascal™ or better with compute capability 6.0+. 
+
+
+### [Step 3: Create the cluster and get credentials](#anchor-create-aks-cluster)
+
+Once you verify that you are allowed to use the necessary VM sizes in your preferred location, now its time to create a managed kubernetes cluster, namely an AKS cluster. The process is pretty simple. Also, after you successfully deploy a cluster with a node-pool of some nodes, you will be able to run workers as pods on the kubernetes cluster using [dask-kubernetes](https://github.com/dask/dask-kubernetes).
+
+- Anyway, first, run the following command to create a AKKS cluster running the latest kubernetes version. It will take a few minutes before it completes. Grab a coffee :coffee: :coffee: .
```

Comment:
```suggestion
- Let's first run the following command to create a AKS cluster using the latest kubernetes version. It will take a few minutes before it completes. Grab a coffee :coffee: :coffee: .
```